tools/scripts/changelog.sh

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/bash
+#
+# This script:
+# - updates the 'Next version' in breaking-changes.md to match the specified version
+# - add back the 'Next version' section, with template
+
+set -eou pipefail
+
+# Ugly but needed for MacOS. Sed on MacOS fails when calling sed --help with an illegal argument exception.
+# Exploit this to check if the user has a bad sed version and exit early.
+if ! sed --help >/dev/null 2>&1; then
```

Comment:
Maybe we could use an alias here if `MacOS then use `gsed` otherwise `sed`?

Reply:
this was not needed, as the way sed is used works on both Linux and MacOS. So I removed it.